code/datums/spells/bloodcrawl.dm

```diff
@@ -32,3 +32,213 @@
 		return
 	revert_cast()
 	to_chat(user, "<span class='warning'>You are unable to blood crawl!</span>")
+
+
+//Travel through pools of blood. Slaughter Demon powers for everyone!
+#define BLOODCRAWL     1
+#define BLOODCRAWL_EAT 2
+
+/mob/living/proc/phaseout(var/obj/effect/decal/cleanable/B)
```

Comment:
Moving this makes it insanely tedious for us to tell what you did, in fact change. YOu're going to have to point out everything you changes in these procs that you moved.